cfpb/cfgov-refresh
# Careers: Add JobPosting structured data to job pages

We have temporarily added hard-coded location data on job posting pages to run a quick job aggregator test before our current set of postings close early next week. This PR adds JobPosting structured data for testing in Google's search results using the same hard-coded location data.


## Additions

- JobPosting structured data section on vacancy announcement pages

## Testing

1. Navigate to one of the pages in the Current Openings list on the [careers landing page](http://localhost:8000/about-us/careers/) (or update the close date for a job posting page in admin if there are no open jobs)

2. View job page's source and verify that jobPosting structured data is included. To test the structured data, copy and paste the page source in [Google's structured data testing tool](https://search.google.com/structured-data/testing-tool). It should report some warnings (we aren't including all the recommended fields), but no errors.

## Screenshots
<img width="1152" alt="screen shot 2017-12-13 at 4 43 47 pm" src="https://user-images.githubusercontent.com/778171/33969898-0440d3da-e025-11e7-8f4f-334d0e2f8e24.png">

<img width="1176" alt="screen shot 2017-12-13 at 4 46 22 pm" src="https://user-images.githubusercontent.com/778171/33969950-4d5a89c6-e025-11e7-8d4f-5f80e01a8817.png">



## Checklist
